packages/compass-web/scripts/sync-dist-to-mms.js

```diff
@@ -58,11 +59,42 @@ const webpackWatchProcess = child_process.spawn('npm', ['run', 'watch'], {
   stdio: 'inherit',
 });
 
+const failProofRunner = () =>
+  new (class FailProofRunner extends Array {
+    append(...fns) {
+      this.push(...fns);
+      return this;
+    }
+
+    run() {
+      const errors = this.map((f) => {
+        try {
+          f();
+        } catch (e) {
+          return e;
+        }
+      }).filter((e) => e);
+
+      if (errors.length) {
+        fs.writeSync(
+          process.stdout.fd,
+          util.inspect(errors, { depth: 20 }) + '\n'
+        );
+      }
+
+      return errors.length;
+    }
+  })();
+
 function cleanup(signalName) {
-  distWatcher.close();
-  webpackWatchProcess.kill(signalName);
-  fs.cpSync(tmpDir, destDir, { recursive: true });
-  fs.rmSync(tmpDir, { recursive: true, force: true });
+  const errorCount = failProofRunner()
```

Comment:
I was getting an error thrown from there I think from cpSync, I figured I'd just make sure we do all the clean up.